### PR TITLE
Feature - use Arcus.Observability Application Insights sink

### LINKED ISSUE #137: Use Application Insights sink from Arcus Observability
Use Application Insights sink from Arcus Observability

### PR DESCRIPTION
**Pull request by stijnmoreels**
Closes #137 
Relates to #138

### CODE REVIEW

File: src/Arcus.Templates.WebApi/Startup.cs [hunk 1]
```diff
@@ -148,9 +145,8 @@ public void ConfigureServices(IServiceCollection services)
 #endif
 
             services.AddHealthChecks();
-#if ExcludeCorrelation
-#else
-            services.AddCorrelation();
+#if ExcludeCorrelation == false
+            services.AddHttpCorrelation();
```
Comment (stijnmoreels) [inline src/Arcus.Templates.WebApi/Startup.cs:149]: This change is actually a a "fix" for the correlation.

Comment (tomkerkhove) [reply src/Arcus.Templates.WebApi/Startup.cs:149]: Thanks!

File: src/Arcus.Templates.ServiceBus.Topic/Program.cs [hunk 2]
```diff
@@ -4,19 +4,17 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-#if ExcludeSerilog
-#else
+using Serilog.Configuration;
```
Comment (tomkerkhove) [inline src/Arcus.Templates.ServiceBus.Topic/Program.cs:7]: This should be in the `#if (ExcludeSerilog == false)` check, no?

Comment (stijnmoreels) [reply src/Arcus.Templates.ServiceBus.Topic/Program.cs:7]: Yes, saw it just now too.

Comment (stijnmoreels) [reply src/Arcus.Templates.ServiceBus.Topic/Program.cs]: Done that.

**Pull request merged**